scripts/environments/local.env.example

```diff
@@ -145,3 +145,7 @@ export ENABLE_DEV_CODE=false
 
 # If you are deploying the solution with the ability to use the Tabular Data Assistant feature, you can set the following values to configure max file size of csv files to be uploaded.
 export MAX_CSV_FILE_SIZE="20" #default is 20MB
+
+# A value used in terraform deployment to set the expiry of passwords measure in days.
```

Comment:
I wouldn't make this a first class parameter of local.env.. rather just update the known issues to show how to add it "if" needed... 

`export TF_VAR_password_lifetime=365`